scripts/issue/791/deprecate_swapr.py

```diff
@@ -0,0 +1,33 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry, get_registry
+
+REGISTRY = get_registry()
+TECH_OPS = REGISTRY.badger_wallets.techops_multisig
+
+SWAPR_WETH = REGISTRY.sett_vaults.bdxsSwaprWeth
+WBTC_WETH = REGISTRY.sett_vaults.bdxsWbtcWeth
+BADGER_WETH = REGISTRY.sett_vaults.bdxsBadgerWeth
+IBBTC_WETH = REGISTRY.sett_vaults.bdxsIbbtcWeth
+
+
+def deprecate_polygon():
```

Comment:
I would advise you to use `main()` if the operation only requires one script. Otherwise you need to add the name of the function whenever running the script from CLI.

Reply:
In any case, it should be `deprecate_swapr` or something..